app/views/content/welcome/landing/_youre-not-the-only-one-with-questions.html.erb

```diff
@@ -0,0 +1,128 @@
+<section class="landing youre-not-the-only-one-with-questions numbered">
+  <div class="arrow-container top mobile">
+    <%=
+      render ArrowComponent.new(
+        width: 150,
+        height: 200,
+        x1: 0,
+        y1: 0,
+        x2: 0.9,
+        y2: 0.9,
+        cx: 0.7,
+        cy: 0.2,
+        arrow_size: 0.06
+      )
+    %>
+  </div>
+
+  <div class="arrow-container top desktop">
+    <%=
+      render ArrowComponent.new(
+        width: 300,
+        height: 100,
+        x1: 0,
+        y1: 0,
+        x2: 0.9,
+        y2: 0.9,
+        cx: 0.3,
+        cy: 1,
+        arrow_size: 0.05
+      )
+    %>
+  </div>
+
+  <%= image_pack_tag("media/images/content/welcome-guide/girl-with-raised-hand.jpg") %>
+
+  <div class="section-number">
+    <span>04</span>
+  </div>
+
+  <div class="text">
+    <h2 class="blue">You're not the only one with questions</h2>
+
+    <p class="blue">
```

Comment:
This paragraph in class="blue" needs removing and replacing with the following text:

We know you’re sure to have questions - maybe even a few concerns – about teaching. It’s only natural. But you’re not alone. It’s a big decision and every year, thousands of people like you ask the same questions.